db/blip_handler.go

```diff
@@ -304,7 +305,11 @@ func (bh *blipHandler) handleSubChanges(rq *blip.Message) error {
 
 		defer func() {
 			bh.changesCtxCancel()
-			bh.activeSubChanges.Set(false)
+			collectionCtx, err := bh.collections.get(bh.collectionIdx)
+			if err != nil {
+				base.WarnfCtx(bh.loggingCtx, "Error getting collectionContext on stopping subChanges request: %w", err)
+			}
```

Comment:
Nit: Is it not possible to use `collectionCtx` or `collectionCtx.activeSubChanges` from the outer scope (line 252) to avoid having to get twice and deal with potential error handling?

Reply:
This is already defined in this function outside this goroutine, removed duplicate.